core/templates/pages/error-pages/error-page.controller.ts

```diff
@@ -41,7 +41,7 @@ angular.module('oppia').directive('errorPage', [
           };
           ctrl.$onInit = function() {
             ctrl.oopsMintImgUrl = UrlInterpolationService.getStaticImageUrl(
-              '/general/oops_mint.png');
+              '/general/oops_mint.webp');
```

Comment:
Will this file work with browsers that do not support WebP?

Reply:
added picture tag in the html